tap/identity-handlers/tyk_handler.go

```diff
@@ -133,6 +135,7 @@ func (t *TykIdentityHandler) CreateIdentity(i interface{}) (string, error) {
 	accessRequest := SSOAccessData{
 		ForSection: thisModule,
 		OrgID:      t.profile.OrgID,
+		EmailAddress: "ssoSession@ssoSession.com",
```

Comment:
Maybe it make sense to pass real user email here? You have access to User object though `i.(goth.User)`, and getting its email can look like:

email := i.(goth.User).Email
if email == "" {
   email = "ssoSession@ssoSession.com"
}

Reply:
I understand that but the Email field is empty when I used pingid (the .createIdentity prints that info), so I have set this value somewhere and that's on the provider level (and different for social, ldap and proxy). Will do for next release